### ISSUE TITLE
[Portfolio] Transactions

### ISSUE THREAD

**Issue by mitchellchou** [oursky/likedao#110]

**Comment by ivanng-oursky** [comment]
@mitchellchou 

> I can view the raw chain data of the transaction(open new tab)

do you mean something like this? No styled applied at all?

https://reqbin.com/echo/get/json
![image.png](https://zube.io/files/oursky/16d549075871a7858e9f305691f9c36b-image.png)

**Comment by mitchellchou** [comment]
@ivanng-oursky 

Yes, take current dao.like.co for example, it just links to the rpc API: https://mainnet-node.like.co/cosmos/tx/v1beta1/txs/130C59FAC1746330742956044C1242713DDD6349E2D3238058E243B0AD024A36

![image](https://user-images.githubusercontent.com/62210372/177250726-fcaf7215-4b82-4655-93c6-697cba32e766.png)
